daprdocs/content/en/operations/components/component-scopes.md

```diff
@@ -89,7 +87,6 @@ apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
   name: statestore
-  namespace: production
```

Comment:
Need to revert this one, since production namespace is used in the example description